src/kernel.cr

```diff
@@ -27,24 +27,24 @@ def gets(*args)
   STDIN.gets(*args)
 end
 
-# Reads a line from STDIN. See `IO#read_line`.
+# Reads a line from STDOUT. See `IO#read_line`.
```

Comment:
Oh but actually that's not true. stdout is where things are written to (in shell what gets redirected for `a` in `a | b` and `a > ./file`), stdin is where things are read from (in shell what gets redirected for `b` in `a | b` and `b < ./file`).